Rest/com.microsoft.alm.client/src/main/java/com/microsoft/visualstudio/services/webapi/patch/json/JsonPatchOperation.java

```diff
@@ -30,7 +30,7 @@ public void setOp(final Operation op) {
      * The path for the operation
      */
      public Operation getPath() {
```

Comment:
Shouldn't the return type of the method change to string if you are going to return path?